lib/firefox/webdriver/firefoxDelegate.js

```diff
@@ -26,7 +26,20 @@ class FirefoxDelegate {
     this.hars = [];
   }
 
-  async onStartIteration() {}
+  async onStartIteration(runner) {
+    if (this.firefoxConfig.geckoProfiler) {
+      let chosenFeatures = this.firefoxConfig.geckoProfilerFeatures.split(',');
+      let featureString = '["' + chosenFeatures.join('","') + '"]';
+
+      let chosenThreads = this.firefoxConfig.geckoProfilerThreads.split(',');
+      let threadString = '["' + chosenThreads.join('","') + '"]';
+
+      // use a 1ms sampling interval and a max buffer of 5000000 entries ~450MB
```

Comment:
We're going to want to tweak these -- do it now, or be ready to follow up :). Especially relevant on Android...

Reply:
Good idea, adding a new option for interval and buffersize.